instance/models/openedx_instance.py

```diff
@@ -146,6 +147,21 @@ def studio_domain(self):
             return self.internal_studio_domain
 
     @property
+    def studio_domain_nginx_regex(self):
+        """
+        Regex that matches either the internal or the external Studio URL.
+
+        This is exclusively meant for the Ansible variable CMS_HOSTNAME to configure the nginx
+        server_name regex to match the Studio domains.
+        """
+        if self.external_studio_domain:
+            domains = [self.external_studio_domain, self.internal_studio_domain]
+        else:
+            domains = [self.internal_studio_domain]
+        choices = '|'.join(map(re.escape, domains))  # pylint: disable=bad-builtin
```

Comment:
@antoviaque I can't speak for Sven but the reason I didn't point this out in my review is that I couldn't think of a reason to prefer a list comprehension in this case (`map` is lazy in Python 3 and the code maps a built-in function as opposed to a `lambda`; `map(re.escape, domains)` doesn't seem less readable than `[re.escape(d) for d in domains]` to me). On the other hand, I am in favor of keeping the number of `# pylint: ...` comments to a minimum, so I wouldn't be opposed to changing this when we get a chance :)